### PR TITLE
Se/code organization

### PR DESCRIPTION
**Pull request by BenG49**

### CODE REVIEW

File: src/main/java/com/stuypulse/robot/commands/amper/AmperScoreAmp.java
Comment (anivanchen) [inline src/main/java/com/stuypulse/robot/commands/amper/AmperScoreAmp.java:1]: Rename to AmperScoreAmpRoutine.

File: src/main/java/com/stuypulse/robot/commands/amper/AmperScoreTrap.java
Comment (anivanchen) [inline src/main/java/com/stuypulse/robot/commands/amper/AmperScoreTrap.java:1]: Rename to AmperScoreTrapRoutine.

File: src/main/java/com/stuypulse/robot/commands/amper/AmperWaitToHeight.java
Comment (anivanchen) [inline src/main/java/com/stuypulse/robot/commands/amper/AmperWaitToHeight.java:1]: Rename to AmperToHeight.

File: src/main/java/com/stuypulse/robot/commands/climber/ClimberToHeight.java
Comment (anivanchen) [inline src/main/java/com/stuypulse/robot/commands/climber/ClimberToHeight.java:1]: Delete this redundant command.

File: src/main/java/com/stuypulse/robot/commands/climber/ClimberWaitForHeight.java
Comment (anivanchen) [inline src/main/java/com/stuypulse/robot/commands/climber/ClimberWaitForHeight.java:1]: Rename to ClimberToHeight.

File: src/main/java/com/stuypulse/robot/commands/conveyor/ConveyorToAmp.java [hunk 1]
```diff
@@ -26,6 +26,7 @@ public ConveyorToAmp() {
     public void execute() {
         conveyor.toAmp();
         intake.acquire();
+        amper.score();
```
Comment (anivanchen) [inline src/main/java/com/stuypulse/robot/commands/conveyor/ConveyorToAmp.java:29]: Shouldn't this call `amper.intake()` instead?

File: src/main/java/com/stuypulse/robot/constants/Motors.java [hunk 2]
```diff
@@ -45,12 +45,12 @@ public interface Shooter {
...
     }
 
     public interface Climber {
-        CANSparkMaxConfig LEFT_MOTOR = new CANSparkMaxConfig(false, IdleMode.kBrake, 80);
-        CANSparkMaxConfig RIGHT_MOTOR = new CANSparkMaxConfig(false, IdleMode.kBrake, 80);
+        CANSparkMaxConfig LEFT_MOTOR = new CANSparkMaxConfig(false, IdleMode.kBrake);
+        CANSparkMaxConfig RIGHT_MOTOR = new CANSparkMaxConfig(false, IdleMode.kBrake);
     }
     
     public static class TalonSRXConfig {
```
Comment (anivanchen) [inline src/main/java/com/stuypulse/robot/constants/Motors.java]: Remove unused `TalonSRXConfig` and `VictorSPXConfig`.

File: src/main/java/com/stuypulse/robot/subsystems/climber/ClimberImpl.java [hunk 3]
```diff
@@ -34,6 +34,9 @@ protected ClimberImpl() {
         rightMotor = new CANSparkMax(Ports.Climber.RIGHT_MOTOR, MotorType.kBrushless);
         leftMotor = new CANSparkMax(Ports.Climber.LEFT_MOTOR, MotorType.kBrushless);
 
+        Motors.Climber.LEFT_MOTOR.configure(leftMotor);
```
Comment (anivanchen) [inline src/main/java/com/stuypulse/robot/subsystems/climber/ClimberImpl.java]: Configure motors at the end of the constructor.

File: src/main/java/com/stuypulse/robot/subsystems/shooter/ShooterImpl.java [hunk 4]
```diff
@@ -27,16 +28,16 @@ public ShooterImpl() {
         leftMotor = new CANSparkMax(Ports.Shooter.LEFT_MOTOR, MotorType.kBrushless);
         rightMotor = new CANSparkMax(Ports.Shooter.RIGHT_MOTOR, MotorType.kBrushless);
 
+        Motors.Shooter.LEFT_SHOOTER.configure(leftMotor);
```
Comment (anivanchen) [inline src/main/java/com/stuypulse/robot/subsystems/shooter/ShooterImpl.java]: Configure motors at the end of the constructor.

File: src/main/java/com/stuypulse/robot/subsystems/swerve/modules/SwerveModuleImpl.java [hunk 5]
```diff
@@ -72,10 +72,14 @@ public class SwerveModuleImpl extends SwerveModule {
     public SwerveModuleImpl(String id, Translation2d offset, int driveID, Rotation2d angleOffset, int turnID, int encoderID) {
         super(id, offset);
         
-        this.angleOffset = angleOffset;        
+        this.angleOffset = angleOffset;
+        
         driveMotor = new CANSparkMax(driveID, MotorType.kBrushless);
         turnMotor = new CANSparkMax(turnID, MotorType.kBrushless);
 
+        Motors.Swerve.DRIVE_CONFIG.configure(driveMotor);
```
Comment (anivanchen) [inline src/main/java/com/stuypulse/robot/subsystems/swerve/modules/SwerveModuleImpl.java]: Configure motors at the end of the constructor.

**Review by anivanchen — CHANGES_REQUESTED** [review]

File: src/main/java/com/stuypulse/robot/commands/conveyor/ConveyorToAmp.java (hunk 1 above)
Comment (anivanchen) [reply src/main/java/com/stuypulse/robot/commands/conveyor/ConveyorToAmp.java:29]: I just reread what `amper.intake()` does. It doesn't need to exist? We're never going to use it.

Comment (BenG49) [reply src/main/java/com/stuypulse/robot/commands/conveyor/ConveyorToAmp.java:29]: I think we leave it, maybe delete amperintake command

**Pull request merged**